### PR TITLE
read component version from env variable

### PR DESCRIPTION
**Pull request by jschicktanz**
**How to categorize this PR?**

/area delivery
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

**Comment by jschicktanz** [comment]
@timebertt @AndreasBurger

File: hack/.ci/component_descriptor [hunk 1]
```diff
@@ -19,14 +19,13 @@ if [[ -f ${resources_file} ]]; then
   echo "${tmp_cd}"
 
   # read component component version.
```
Comment (timebertt) [inline hack/.ci/component_descriptor]:
```suggestion
  # read the component version.
```

**Review by timebertt — COMMENTED** [review]
Please see the other comments by @ialidzhikov in https://github.com/gardener/gardener/pull/5818

File: hack/.ci/component_descriptor (hunk 1 above)
Comment (jschicktanz) [reply hack/.ci/component_descriptor]: fixed

**Comment by jschicktanz** [comment]
thanks, I incorporated the remaining open comments from this PR and #5818

**Review by timebertt — APPROVED** [review]
/lgtm

**Pull request merged**